### ISSUE TITLE
roxygenize

### ISSUE THREAD

**Issue by jenessalemon** [jenessalemon/R-World-Package#2]
@willpearse I ran:

install.packages('roxygen2')
roxygenize("/Users/jimblotter/Desktop/Grad School/Programming_for_Biologists/r-world-jenessalemon/plants.R") 

Which is the first three lines of plants.R. I ran the whole file, and nothing happened. I have been using #' notation and I'm not sure why the other folders in my package didn't automatically generate with roxgenize. I didn't get an error, but the files are still empty. Any ideas as to what is going on? I would love to skype you tomorrow during office hours.

**Comment by jenessalemon** [comment]
I just messed around and ran it again and this time I got an error:
> roxygenize("/Users/jimblotter/Desktop/Grad School/Programming_for_Biologists/r-world-jenessalemon/plants.R")
**Error: could not find function "roxygenize"**

Am I loading the package correctly?

**Comment by willpearse** [comment]
No. You haven't loaded the package. You need to write ```library(roxygen2)```.

You also need to create a package before you can roxygenize. Follow my instructions in the lecture.

**Comment by jenessalemon** [comment]
Ok, it would be great if you could upload the lecture recording from Monday, because I've read the lecture slides multiple times and I'm struggling. Also the handout says we need to first check the package, then build the package, but the lecture slides say the opposite.
